spec/models/topic_converter_spec.rb

```diff
@@ -183,7 +183,7 @@
 
       it "invites only users with regular posts" do
         post2 = Fabricate(:post, topic: topic)
-        whipser_post = Fabricate(:post, topic: topic, post_type: Post.types[:whisper])
+        whisper_post = Fabricate(:post, topic: topic, post_type: Post.types[:whisper])
```

Comment:
nitpick, but this can also just be 
```suggestion
        Fabricate(:post, topic: topic, post_type: Post.types[:whisper])
```